stingray/tests/test_crossspectrum.py

```diff
@@ -376,6 +376,11 @@ def test_rebin_log(self):
         assert type(new_cs) == type(self.cs)
         new_cs.time_lag()
 
+    def test_rebin_log_returns_complex_values(self):
+        # For now, just verify that it doesn't crash
+        new_cs = self.cs.rebin_log(f=0.1)
+        assert isinstance(new_cs.power[0], np.complex)
```

Comment:
I'm curious if there are strange side effects that make the error bars double _before_ rebinning? Could you test also the error bars being complex?

Reply:
Test added.

Reply:
Unless there are other comments, I think this is ready to merge. I don't know what coveralls is doing; I looked at couldn't find any pertinent lines of code where coverage actually decreased.

Reply:
It might be that the total lines of code decreased, even without adding new uncovered lines? Anyways, merging!